.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Misschien moet er meer uit de `.idea` folder worden ge-gitignored. Nu stuurt die ook wel vage bestanden mee, nadat ik `npm i`, `npm run dev` of `composer i` heb gedaan.

Reply:
Is het niet handiger om gewoon de hele .idea map in gitignore te doen?

Reply:
Ik heb onderzoek gedaan waarom deze bestanden elke keer weer opnieuw worden gegenereerd. De bestanden worden aangemaakt om versies te noteren. Dit kan voor merge-conflicts zorgen, omdat bij het gebruik van `composer i` of `npm run dev` kunnen de bestanden worden aangepast. 

Aangezien het automatisch wordt gegenereerd in iedereens project, kan het geen kwaad om die niet te uploaden naar Github. Ook zorgt het voor geen problemen als de bestanden er niet staan. Het is gewoon een chache dingetje. Daarom heb ik dit nu toegevoegd aan de main .gitignore.

https://www.dev-metal.com/how-to-ignore-remove-idea-folder-from-git-in-phpstorm/